screens/HomeScreen.js

```diff
@@ -1,21 +1,19 @@
 import React from 'react';
-import { View, StyleSheet, Button} from 'react-native';
+import { View, StyleSheet, Button } from 'react-native';
 import useStatusBar from '../hooks/useStatusBar';
 import { logout } from '../components/Firebase/firebase';
-import {NavigationContainer} from '@react-navigation/native';
-import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
-import {Image} from 'react-native';
-import Colours from '/Users/sandra/Maiagram-expo-firebase/screens/res/Colours.js'
```

Comment:
Importing via an absolute path causes errors when other team members try to run it